### PR TITLE
fix: let the classified-bar scroll with content [FS-1742]

### PR DESCRIPTION
**Pull request by aweiss-dev**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/FS-1742" title="FS-1742" target="_blank">BugFS-1742</a>  [Web] Connection request profile photo overlaps on classified/unclassified banner when scrolling down
  </td></table>
  <br />
 

Problematic behavior:

bug

Fix:

fix

### CODE REVIEW

**Review by tlebon — APPROVED** [review]
still not sure what was wrong with the old one 🙃

**Review by thisisamir98 — APPROVED** [review]

File: src/script/components/ModalComponent.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ const ModalContentStyles: CSSObject = {
   maxHeight: 615,
   overflow: 'hidden',
   overflowY: 'auto',
-  padding: 16,
+  padding: 0,
```
Comment (atomrc) [inline src/script/components/ModalComponent.tsx]: Are you confident this didn't change other modal's styles 😬 ?

File: src/style/modal/user-modal.less [hunk 2]
```diff
@@ -27,4 +27,17 @@
...
+  }
+
+  .modal__body {
+    padding: 16px;
+
+    .classified-bar {
+      width: calc(100% + 32px);
+      margin-top: 12px;
+    }
+  }
```
Comment (atomrc) [inline src/style/modal/user-modal.less]: Is there anyway we could colocate those styles next to the targeted component in a `Component.styles.ts` file?

File: src/script/components/ModalComponent.tsx (hunk 1 above)
Comment (aweiss-dev) [reply src/script/components/ModalComponent.tsx]: Good question. I need to check 🥶

**Review by atomrc — APPROVED** [review]

**Pull request merged**